pom.xml

```diff
@@ -20,6 +20,12 @@
             <artifactId>okhttp</artifactId>
             <version>4.11.0</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```

Comment:
Yay, testing! Any chance you can add back your tests?

Reply:
yes, done